backend/controller/timeline/events_cron.go

```diff
@@ -0,0 +1,82 @@
+package timeline
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+
+	ftlencryption "github.com/TBD54566975/ftl/backend/controller/encryption/api"
+	"github.com/TBD54566975/ftl/backend/controller/timeline/internal/sql"
+	"github.com/TBD54566975/ftl/backend/libdal"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/model"
+)
+
+type CronScheduledEvent struct {
+	ID            int64
+	DeploymentKey model.DeploymentKey
+	Verb          schema.Ref
+
+	Time        time.Time
+	Duration    time.Duration
+	ScheduledAt time.Time
+	Schedule    string
+	Error       optional.Option[string]
```

Comment:
I think you can embed `CronScheduled` here so you don't have to duplicate these fields.

```go
type CronScheduledEvent struct {
	ID            int64
	CronScheduled
}
```

Assuming they are the same set of fields.

Reply:
This embeds the `CronScheduled` struct under the `CronScheduled` key though, right?

Reply:
Nvm, ignore me; that's true for initialization but not access